contentctl/objects/abstract_security_content_objects/detection_abstract.py

```diff
@@ -74,6 +74,8 @@ class Detection_Abstract(SecurityContentObject):
 
     data_source_objects: list[DataSource] = []
 
+
```

Comment:
nit: functions w/in a class should only have 1 empty line between them

Reply:
Agreed, fixed and pushed.
Resolving